### PR TITLE
1138 helm3 integration tests

### PR DESCRIPTION
**Pull request by glindsell**
#1138

(continued from #1181)

- Updated Integration tests in line with Helm v3

- Added to test README.md

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **gsunner**

If they are not already assigned, you can assign the PR to them by writing `/assign @gsunner` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
@glindsell: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 8f41bd85744520b449d133902421920ad9dacbe6 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@glindsell: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | a0affbf169d8b2fed905dae463b2ffe16588b0a2 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@glindsell: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 5fc07f14ff6fa90088cb946857b0896366de6422 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: operator/go.mod [hunk 1]
```diff
@@ -6,21 +6,16 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.2 // indirect
 	github.com/Azure/go-autorest/autorest/adal v0.7.0 // indirect
 	github.com/go-logr/logr v0.1.0
-	github.com/gogo/protobuf v1.2.1 // indirect
```
Comment (adriangonz) [inline operator/go.mod]: Should we revert the changes to the `go.mod` and `go.sum` dependencies of the `operator`? What triggered these?

Comment (glindsell) [reply operator/go.mod]: Have merged with master and run `go mod tidy`

**Comment by seldondev** [comment]
@glindsell: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | c76b1210b74b19a10105fbccf4c94dbd5cd8f48b | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Fri Dec  6 09:55:08 UTC 2019
The logs for [integration] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1186/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1186 --build=11

**Comment by seldondev** [comment]
@glindsell: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | c76b1210b74b19a10105fbccf4c94dbd5cd8f48b | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Fri Dec  6 14:48:32 UTC 2019
The logs for [pr-build] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1186/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1186 --build=12

**Comment by seldondev** [comment]
Fri Dec  6 14:48:37 UTC 2019
The logs for [lint] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1186/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1186 --build=13

**Comment by seldondev** [comment]
Fri Dec  6 15:00:22 UTC 2019
The logs for [integration] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1186/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1186 --build=14

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1186#issuecomment-562641321" title="Approved">ryandawsonuk</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [ryandawsonuk]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
